foxy/ci-nightly-performance.yaml

```diff
@@ -0,0 +1,1540 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  CMAKE_PREFIX_PATH: '/opt/ros/melodic:$CMAKE_PREFIX_PATH'
```

Comment:
Foxy is targeting `noetic`, not `melodic`. 3 more environment variables have the same issue, as well as the `install_packages` list.

This part of the config should closely resemble this: https://github.com/ros2/ros_buildfarm_config/blob/ros2/foxy/ci-nightly-extra-rmw-release.yaml

Reply:
fixed